cucumber/authenticators_jwt/features/authn_jwt_token_schema.feature

```diff
@@ -0,0 +1,70 @@
+Feature: JWT Authenticator - Token Schema
+
+  Tests checking mandatory claims and claims mapping
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group hosts
+
+      - !permit
+        role: !group hosts
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+
+  Scenario: ONYX-10471 - Mandatory Claims Without Claims Mapping. Single Mandatory - 200 OK
+    Given I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    And I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: mandatory-claims
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/ref: valid
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "mandatory-claims" variable to value "ref"
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I permit host "myapp" to "execute" it
+    And I issue a JWT token:
+    """
+    {
+      "ref":"valid",
+      "host":"myapp"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then host "myapp" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
```

Comment:
add log which validate the mandatory claim

Reply:
The error tests are enough for me to be sure they are valdiated. So this log check not really needed in my opinion and will just hurt the readiblity of the test

Reply:
Just because it is a new feature 

The problem is: this test will also passed successfully on previous version without the new code